src/Numerical/HBLAS/BLAS/Internal.hs

```diff
@@ -182,6 +186,35 @@ gemvAbstraction gemvName gemvSafeFFI gemvUnsafeFFI constHandler = gemv
                          (fromIntegral bstride) betaPtr cp (fromIntegral cstride)
 
 
+{-# NOINLINE gerAbstraction #-}
+gerAbstraction :: (SM.Storable el, PrimMonad m)
+               => String
+               -> GerFunFFI el
+               -> GerFunFFI el
+               -> forall orient . GerFun el orient (PrimState m) m
+gerAbstraction gerName gerSafeFFI gerUnsafeFFI = ger
+    where
+      shouldCallFast :: Int -> Int -> Bool
+      shouldCallFast m n = flopsThreshold >= (fromIntegral m :: Int64)
```

Comment:
@archblob  derp, should this have been a `<=` instead? ie we should be calling the fast code IF `m*n` is *LESS* than `flopsThreshold`? shows what I get for merging a PR before doing a close reading